src/sentry/search/events/builder/discover.py

```diff
@@ -320,7 +320,7 @@ def load_config(
         Mapping[str, Callable[[Direction], OrderBy]],
     ]:
         self.config: DatasetConfig
-        if self.dataset in [Dataset.Discover, Dataset.Transactions, Dataset.Events]:
+        if self.dataset in [Dataset.Discover, Dataset.Transactions, Dataset.Events, Dataset.IssuePlatform]:
```

Comment:
Not entirely sure this is the right place to add it.

Reply:
seems right. are you seeing the data returned from the endpoint in testing?

Reply:
Yeah, the test seems to be returning it fine